client/internal/httpsender.go

```diff
@@ -97,6 +103,10 @@ func (h *HTTPSender) makeOneRequestRoundtrip(ctx context.Context) {
 	if err != nil {
 		return
```

Comment:
not scope of this PR but just noticed this - shouldn't we at least log the error?